### PR TITLE
feat: Blobby always use offset store

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Once we have tested it out and are happy then we can remove the old code paths

## Changes

* Removes the flag
* Removes an old unused util
* Fixes the tests

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts [hunk 1]
```diff
@@ -473,7 +461,7 @@ describe.each([[true], [false]])('ingester with consumeOverflow=%p', (consumeOve
             })
         })
 
-        describe('simulated rebalancing', () => {
+        describe('simulated rebalanng', () => {
```
Comment (frankh) [inline plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts:464]: nit: typo

Comment (benjackwhite) [reply plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts:464]: wat?

**Review by pauldambra — APPROVED** [review]
👍 

(Aside where can I go to read about what the offset store does 🙈)

**Comment by benjackwhite** [comment]
> 👍
> 
> (Aside where can I go to read about what the offset store does 🙈)

@pauldambra It's not well documented. The main difference is that we are telling the SDK what offsets _should_ be commited rather than actually doing the committing ourselves.

rdkafka then manages ensuring that we only commit to valid partitions, generating the right values etc.

**Pull request merged**